string responderId
float32 precisionRangeM
float32 precisionRangeErrEst